acrobat/scripts/scripts.js

```diff
@@ -250,3 +267,8 @@ const bowserReady = setInterval(() => {
     window.dispatchEvent(bowserIsReady);
   }
 }, 100);
+
+const { default: ccTranslations } = await import('./ccTranslations.js');
+document.addEventListener('milo:deferred', () => {
+  ccTranslations();
```

Comment:
you should pass the elements in this function so you don't have to search the DOM again